File: CSharp/IP2C/Record.cs
using System;

namespace IP2C
{
    public class Record
    {
        public Record(uint ipFrom, uint ipTo, string registry, double assigned, string ctry, string cntry, string country)
        {
            this.ipFrom = ipFrom;
            this.ipTo = ipTo;
            this.registry = registry;
            this.assigned = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc).AddSeconds(assigned);
            this.ctry = ctry;
            this.cntry = cntry;
            this.country = country;
        }
        public uint ipFrom { get; }
        public uint ipTo { get; }
        public string registry { get; }
        public DateTime assigned { get; }
        public string ctry { get; }
        public string cntry { get; }
        public string country { get; }

    }
}

File: Java/Record.java
import java.util.Date;

class Record{
    private long ipFrom;
    private long ipTo;
    private String registry;
    private Date assigned;
    private String ctry;
    private String cntry;
    private String country;

    public Record(long ipFrom, long ipTo, String registry, Date assigned, String ctry, String cntry, String country){
        this.ipFrom = ipFrom;
        this.ipTo = ipTo;
        this.registry = registry;
        this.assigned = assigned;
        this.ctry = ctry;
        this.cntry = cntry;
        this.country = country;
    }

    public Record(String ipFrom, String ipTo, String registry, String assigned, String ctry, String cntry, String country){
        this(Long.parseLong(ipFrom), Long.parseLong(ipTo), registry, new Date(Long.parseLong(assigned)), ctry, cntry, country);
    }

    public long getIpFrom(){
        return ipFrom;
    }

    public long getIpTo(){
        return ipTo;
    }

    public String getRegistry(){
        return registry;
    }

    public Date getAssigned(){
        return assigned;
    }

    public String getCtry(){
        return registry;
    }

    public String getCntry(){
        return cntry;
    }

    public String getCountry(){
        return country;
    }

    public void setIpFrom(long ipFrom){
        this.ipFrom = ipFrom;
    }

    public void setIpTo(long ipTo){
        this.ipTo = ipTo;
    }

    public void setRegistry(String registry){
        this.registry = registry;
    }

    public void setAssigned(Date assigned){
        this.assigned = assigned;
    }

    public void setCtry(String ctry){
        this.ctry = ctry;
    }

    public void setCntry(String cntry){
        this.cntry = cntry;
    }

    public void setCountry(String country){
        this.country = country;
    }
}